src/main/java/edu/ucsb/cs156/courses/controllers/CourseOverTimeController.java

```diff
@@ -35,7 +35,8 @@ public ResponseEntity<String> search(
         @RequestParam String startQtr,
         @RequestParam String endQtr,
         @RequestParam String subjectArea,
-        @RequestParam String courseNumber
+        @RequestParam String courseNumber,
```

Comment:
I'm not sure I understand why we are adding this parameter to the controller endpoint, when that parameter is not being passed to the query.

The course results use the other four parameters, but not this one.  So this seems misleading.

Can you help me understand your intention here?

The code that actually gets this courses is this code from a few lines down, and it doesn't do anything with the `location` parameters.

```java
courseResults = convertedSectionCollection.findByQuarterRangeAndCourseId(
            startQtr,
            endQtr,
            makeFormattedCourseId(subjectArea, courseNumber)
);
```

Reply:
Ok my apologizes. I didn't realize how much this change would impact everything. I will go back and rethink how to approach this issue.